model/permission.go

```diff
@@ -636,4 +638,12 @@ func initializePermissions() {
 
 func init() {
 	initializePermissions()
+	ModeratedPermissions = map[string]bool{
+		PERMISSION_ADD_REACTION.Id:                   true,
+		PERMISSION_REMOVE_REACTION.Id:                true,
+		PERMISSION_CREATE_POST.Id:                    true,
+		PERMISSION_MANAGE_PUBLIC_CHANNEL_MEMBERS.Id:  true,
+		PERMISSION_MANAGE_PRIVATE_CHANNEL_MEMBERS.Id: true,
+		// PERMISSION_USE_CHANNEL_MENTIONS:              true,
+	}
```

Comment:
I have a similar data structure in my PR which maps it to the channel moderated permission name 
i.e. maps `add_reactions` to `create_reactions`

Can always be consolidated later but just making a note of it here for later

Reply:
@fm2munsh I was planning to swap this for your data structure.